package.json

```diff
@@ -34,6 +34,11 @@
     "ts-jest": "28.0.5",
     "turbo": "1.7.4"
   },
+  "pnpm": {
+    "overrides": {
+      "gatsby-sharp>sharp": "npm:is-odd-num@1.1.1"
```

Comment:
This package is actually quite large https://packagephobia.com/result?p=is-odd-num

Reply:
I think this is going to lead to more confusion if someone ever tries to add a test for gatsby and expects sharp to work (for example `vc dev` tests)

Reply:
Why `is-odd-num` needed to add `mocha` as a `dependency` instead of a `devDependency` is puzzling.